### DISCUSSION TITLE
Any updates on this front?

### DISCUSSION THREAD

**Discussion by Rocketboy235** [coggy9/HKHacking#13] [General]
Hello! I've been kind of following this over the years but was never really involved. I'm curious how far people have gotten with accessing root control of HK Invoke and if that means we can start customizing it freely or if it is unfortunately still locked down. Thanks.

**Comment by coggy9** [comment]
There hasn't been any real progress past rooting it, unfortunately.

**Reply by spicehoarder** [reply]
What?! Is there a rooting guide? I just assumed we hadn't been able to run arbitrary code yet...

**Reply by coggy9** [reply]
There isn't really a guide, but it is possible. 

Tl;dr: Use [this](https://github.com/coggy9/HKHacking/releases/tag/StockRoot) system image in place of the [stock](https://github.com/coggy9/HKHacking/releases/tag/HarmanFlash) one, then connect over wireless ADB. Might need a reset if you're downgrading.
